sssd_test_framework/utils/local_users.py

```diff
@@ -12,9 +14,20 @@
     "LocalGroup",
     "LocalUser",
     "LocalUsersUtils",
+    "LocalSudoRule",
 ]
 
 
+class SudoRuleDefaults:
+    """
+    Default values for sudo rules.
+    """
+
+    user = "ALL"
```

Comment:
These class constants could be part of the `LocalSudoRules` class. There is no need for a separate class.